services/user.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str | None = None,
+                first_name: str | None = None,
+                last_name: str | None = None) -> None:

```

Comment:
Place each argument on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability